Tasks/MavenV3/maventask.ts

```diff
@@ -121,6 +121,10 @@ if (specifiedJavaHome) {
     tl.setVariable('JAVA_HOME', specifiedJavaHome);
 }
 
+function checkExistance(obj: any): Boolean {
```

Comment:
NIT: type should be boolean, not Boolean. Also, do we really need this function? I feel like !!something or Boolean(something) would be better for readability